tests/cms/models/chat/test_user_chat.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from django.conf import settings
+
+from integreat_cms.cms.models import UserChat
+
+
+@pytest.mark.django_db
+def test_update_mt_budget() -> None:
+    """
+    Test if words are being correctly added to the used MT budget and chat word count
+    """
```

Comment:
I think explicitly mentioning that this test is unit testing a method would help. *(I hope I got the reference right)*
```suggestion
def test_update_mt_budget() -> None:
    """
    Test if words are being correctly added to the used MT budget and chat word count
    in :meth:`~integreat_cms.cms.models.chat.user_chat.Chat.update_mt_budget`
    """
```